### ISSUE TITLE
Create simple view to upload CSV extract to backend

### ISSUE THREAD

**Issue by Jakub-Mikolajczyk-pl** [Xupi/WishList-Compare#3]
Extract will be created from goodreads, in csv file.

**Comment by hakimrabet** [comment]
@Xupi would you explain it more please?

**Comment by Jakub-Mikolajczyk-pl** [comment]
> @Xupi would you explain it more please?

Yeah, I need a simple frontend site that will have a dropzone for CSV extract file from goodreads wishlist, to send it to backend for further processing.
